views/update/overseas-entity-query.html

```diff
@@ -44,7 +44,7 @@ <h1 class="govuk-heading-xl">{{ title }}</h1>
         <div class="govuk-details__text">
           <p class="govuk-body">This is given to an overseas entity when it registers with Companies House. You can find it:</p>
           <ul class="govuk-list govuk-list--bullet">
-            <li>by searching for the entity on the <a class="govuk-link" href="#">Companies House register (opens in a new tab)</a></li>
+            <li>by searching for the entity on the <a class="govuk-link" target="_blank" rel="noopener" href="https://find-and-update.company-information.service.gov.uk/">Companies House register (opens in a new tab)</a></li>
```

Comment:
This is the live url. You should use the CHS_URL environment variable to get the correct url rather than hardcode one.

Reply:
Changes pushed up